### ISSUE TITLE
Add per-protocol user counts to nodeinfo

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy-fed#1509]
Inspired by https://bsky.app/profile/did:plc:fdme4gb7mu7zrie7peay7tst/post/3lbabzmnccs2w

Would be non-standard, but if I namespace the field names, that should hopefully be fine...?

Nodeinfo spec: https://nodeinfo.diaspora.software/protocol.html

**Comment by Tamschi** [comment]
Another option would be to serve the subset counts on the respective subdomains, e.g. `https://fed.brid.gy/nodeinfo.json#usage.users.total` being the total number of Bluesky users whose account is being bridged.

That wouldn't necessarily tell the whole story, since it wouldn't say to which networks they are bridged, but it would work with existing surveying tools. Though, in that case maybe those counts should be excluded from the one that `fed.brid.gy` serves to avoid duplication in aggregate stats.

**Comment by snarfed** [comment]
> Another option would be to serve the subset counts on the respective subdomains, e.g. `https://bsky.brid.gy/nodeinfo.json#usage.users.total` being the total number of Bluesky users whose account is being bridged.

True! That would break up Bridgy Fed in fediverse stats aggregators into its separate protocols, one "instance" per protocol. Which is maybe fine? 🤷 

> The schema at https://nodeinfo.diaspora.software/schema.html says to put software-specific values into the `"metadata"` property.

True! But by software I think they mean implementation, not protocol. Could still work though.

**Comment by Tamschi** [comment]
> True! But by software I think they mean implementation, not protocol. Could still work though.

I mean, "`bridgy-fed`" is an implementation I think 😅
